hbase-server/src/test/java/org/apache/hadoop/hbase/regionserver/TestHRegion.java

```diff
@@ -7364,4 +7367,154 @@ protected HStoreForTesting(final HRegion region,
       return super.doCompaction(cr, filesToCompact, user, compactionStartTime, newFiles);
     }
   }
+
+  @Test(timeout=20000)
+  public void testCloseNoInterrupt() throws Exception {
+    byte[] cf1 = Bytes.toBytes("CF1");
+    byte[][] families = { cf1 };
+
+    Configuration conf = HBaseConfiguration.create();
```

Comment:
Use the config from the HBTU? Maybe new Configuration(TEST_UTIL.getConfiguration())

Reply:
Ok